extract_verbs.sh

```diff
@@ -3,10 +3,97 @@
 set -e
 
 cd ..
-# Download latest Italian Wikipedia dump
-wget http://download.wikimedia.org/itwiki/latest/itwiki-latest-pages-articles.xml.bz2
+# Download latest Wikipedia dump in language of choice
+echo "Enter wikipedia dump language e.g. en OR english"
```

Comment:
This should be replaced by a command line argument.
The script should not be interactive.